tests/php/ORM/DataQueryTest.php

```diff
@@ -466,8 +466,51 @@ public function testMultipleRelationSort()
 
     public function testExistsCreatesFunctionalQueries()
     {
-        $this->assertTrue(ObjectE::get()->exists());
-        $this->assertFalse(ObjectE::get()->where(['"Title" = ?' => 'Foo'])->exists());
-        $this->assertTrue(ObjectE::get()->dataQuery()->groupby('"SortOrder"')->exists());
+        $this->assertTrue(
```

Comment:
I've updated the scenarios here to test both the DataList exist and the DataQuery exists. I've added scenarios for the limit and added explicit message for each assertion.